FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift

```diff
@@ -20,16 +20,26 @@ final class FlowersListPresenter {
     private weak var output: FlowersListPresenterOutput?
     private let interactor: FlowersListInteractorInput
     
+    private var teste: String?
```

Comment:
<!--
  0 failure: 
  1 warning:  Property 'teste' ...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift;
  Line: 23;
-->

- :warning: Property 'teste' is unused